projects/shared-pipes/safe-url.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer, SafeUrl } from '@angular/platform-browser';
+
+@Pipe({
+  name: 'safeUrl'
+})
+export class SafeUrlPipe implements PipeTransform {
+
+  constructor(private domSanitizer: DomSanitizer) {
+  }
+
+  transform(url: string): SafeUrl {
```

Comment:
Why do we need this pipe?

Reply:
we are converting the inputImage file into dataUrl and we cannot directly load an svg image because it creates security issues. The pipe converts the url into safe url. We can also use the safeHtml pipe and just inject the svg into div using innerHtml, that will work too!